staging/src/k8s.io/apimachinery/pkg/conversion/converter.go

```diff
@@ -44,23 +44,16 @@ type Converter struct {
 	generatedConversionFuncs ConversionFuncs
 
 	// Set of conversions that should be treated as a no-op
-	ignoredConversions        map[typePair]struct{}
 	ignoredUntypedConversions map[typePair]struct{}
-
-	// nameFunc is called to retrieve the name of a type; this name is used for the
-	// purpose of deciding whether two types match or not (i.e., will we attempt to
-	// do a conversion). The default returns the go type name.
-	nameFunc func(t reflect.Type) string
```

Comment:
I'm surprised that it's not used. But given it's not, let's also change the NewConverter to not take any parameter if it's always unused.

Reply:
It will change NewConverter's signature. Can we accept this incompatible change?

Reply:
Signature is the same (it still takes a single `NameFunc` argument, it just doesn't do anything with it now).

Reply:
Current implement keep `NewConverter` signature, but ignore args.